modules/serverless-eventing-components.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * serverless-knative-eventing.adoc
+
+[id="serverless-eventing-components_{context}"]
+= Knative Eventing components
+
+A typical Knative Serving deployment on {ServerlessProductName} follows this basic workflow.
+
+. Event producers integrated with {ServerlessProductName} send information about changes in the state of the system to an event source.
```

Comment:
This whole description is rather hard to understand. I would word that differently. From usability perspective KE is easy to integrate event propagation system. It delivers events that are traveling in channels to subscribers, and triggers define to which one given event should go. Subscribers can be Knative Services, and regular Kubernetes services. We can feed in events directly to a broker, with simple HTTP request, or used integrated event producers that supports sources like Kafka or Camel K.